src/pybind/mgr/cephadm/module.py

```diff
@@ -2139,11 +2140,18 @@ def _apply_service_spec(self, spec: ServiceSpec) -> str:
             get_daemons_func=self.cache.get_daemons_by_service,
         ).validate()
 
-        self.log.info('Saving service %s spec with placement %s' % (
-            spec.service_name(), spec.placement.pretty_str()))
+        msg = 'Saving service %s spec with placement %s' % (
+            spec.service_name(),
+            spec.placement.pretty_str())
+        self.log.info(msg)
         self.spec_store.save(spec)
         self._kick_serve_loop()
-        return "Scheduled %s update..." % spec.service_name()
+
+        if used_default_placement:
+            msg = '%s\n' % msg
```

Comment:
What do you think of adding this line independent  of `used_default_placement`?

Reply:
@sebastian-philipp : There are a number of unit tests which requires the exact output of "Scheduled <service> update..." in order to know that an "apply" command  has been launched properly.
This is why i have added this "exception" for "use cases" where default settings are going to be used

Reply:
Hm. Should not be super hard to change the unit tests I think